storage/innobase/os/os0file.cc

```diff
@@ -794,12 +794,12 @@ os_file_punch_hole_posix(
 /** Retrieves the last error number if an error occurs in a file io function.
 The number should be retrieved before any other OS calls (because they may
 overwrite the error number). If the number is not known to this program,
-the OS error number + 100 is returned.
+the OS error number + 200 is returned.
```

Comment:
I would suggest that instead of writing the value of constant, be more flexible and write `OS_FILE_ERROR_MAX` as a static constant may be changed  in `storage/innobase/include/os0file.h`.

Reply:
Ok, I will change it. sorry for deleting/restoring branch and closing/reopening PR, it was an accident...

Reply:
One more thing please. 
Since this is first done on `10.2` can you please modify this PR and make use of `10.2` target instead. No need to open new PR, if you need help let me know.

Reply:
Yes, no problem. Done.